CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.0.1] - 2021-02-22
+### Added
+- First release of Lab Results Adaptor
+- NHSACK is not produced
+- Limited set of resources produced - Requester (FHIR Practitioner resource)
+
+### Changed
+n/a
+
+### Removed
+n/a
+
+### Known Issues
+- Information Disclosure [Medium Severity][https://snyk.io/vuln/SNYK-JAVA-COMGOOGLEGUAVA-1015415] in com.google.guava:guava@29.0-jre introduced by com.google.guava:guava@29.0-jre
```

Comment:
I don't think we're using Guava in the adaptor or are these known issues for transitive dependencies?

Reply:
it's indirect - that's one of the dependencies dependency